packages/cerebral-provider-firebase/src/helpers.js

```diff
@@ -110,3 +110,7 @@ export function convertObjectWithTemplates (obj, resolve) {
     return convertedObject
   }, {})
 }
+
+export function noop () {
+  return function () {}
+}
```

Comment:
Ouch. Smth wrong here. It is definitely not a noop. Why it returns empty function? It should be empty function on its own.